05/Menu.js

```diff
@@ -4,15 +4,29 @@ import ReactDOM from 'react-dom';
 
 import MenuItem from '../03/MenuItem';
 
-const Menu = () => {
-	const arr = [<MenuItem text='strona główna' url='/' />, <MenuItem text='www' url='/www' />];
-	const jsx = <ul>{arr}</ul>;
-	return jsx;
-};
+// const Menu = () => {
+// 	const arr = [<MenuItem text='strona główna' url='/' />, <MenuItem text='www' url='/www' />];
+// 	const jsx = <ul>{arr}</ul>;
+// 	return jsx;
+// };
 
-MenuItem.PropTypes = {
-	text: PropTypes.string.isRequired,
-	url: PropTypes.string.isRequired,
-};
+const Menu = (props) => {
+    const { items } = props;
+    return (
+        <ul> {
+            items.map(
+                item => {
```

Comment:
Destrukturyzajce można też od razu tutaj zrobic tj. zamiast `item` to ({text, url})